src/influencetx/legislators/templates/legislators/legislator_detail.html

```diff
@@ -10,9 +10,29 @@
 {% endblock css %}
 
 {% block content %}
-
+<script>
+  var sharePage = "https://app.influencetexas.com/legislators/legislators/{{ legislator.id }}/";
+</script>
 <div class="summary-info text-center mt-3 mb-4">
-  <img src="{{ legislator.photo_url }}"/>
+  <div class="flexcontainer-leg-share">
+    <img src="{{ legislator.photo_url }}">
+      <div class="itemright-leg-share">
+          <a href="" onclick="window.open('https://www.facebook.com/share.php?u=' + 
```

Comment:
I was able to fix the links by using the following code:
`<a href="https://www.facebook.com/share.php?u={{ request.build_absolute_uri }}" target="_blank">
      <span class='symbol'>circlefacebook</span>
    </a>`
This removes the need to use onclick and an anchor, as well as removed the need to generate the "sharePage" manually.
The horizontal display looks great on the home page, but for the detail pages it was cutting into the images/header info, so vertically orienting them by adding a couple of "<p>" tags fixed that.
There were also some merge conflicts with some other updates that were made to the templates, so I'm wondering you would like to resolve those?